components/raftstore/src/store/util.rs

```diff
@@ -1200,10 +1200,11 @@ impl RegionReadProgressRegistry {
     ) -> Vec<u64> {
         let mut regions = Vec::with_capacity(leaders.len());
         let registry = self.registry.lock().unwrap();
+        let now = Some(Instant::now());
```

Comment:
As mentioned in the above comments, would it introduce much performance overhead considering the frequency of checking leaders, or should we use `now_coarse()` instead?

Reply:
I believe this is acceptable by estimating the impact: If there are 1M active regions, interval of check leader is 5s, a `now()` cost ~20ns in my testing Linux environment. That would be 0.02s/5s = <1% CPU usage.
`now_coarse` sounds good.